.github/workflows/analyze_and_test.yml

```
@@ -0,0 +1,50 @@
+name: Pull Request - Code Analysis and Unit Tests
+on:
+  pull_request:
+    branches: [ dev ]
+env:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_CLI_TELEMETRY_OPTOUT: true
+jobs:
+  build_test:
+    runs-on: ubuntu-latest
+    name: Build & Test
```

Just to be in line with TC:

```suggestion
    name: Build and Test
```